Bottom: d22334bfbdb8b27c8a2b7e6c46db994cd233eb3e
Top:    d5ddf6fcafb96931c00e0d706ac3320a55acb9f9
Author: scott snyder <sss@karma>
Date:   2020-02-12 08:45:25 -0500

Don't override installation path.


---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a421b1e..200d622 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@ OPTION( FCAL_USE_Marlin  " Build Marlin Processors" True )
 OPTION( FCAL_USE_DD4hep  " Build With DD4hep support" True )
 OPTION( INSTALL_DOC "Set to OFF to skip build/install Documentation" OFF )
 
-SET( CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR} )
+#SET( CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR} )
 
 FIND_PACKAGE( ILCUTIL 1.3.0 REQUIRED COMPONENTS ILCSOFT_CMAKE_MODULES )
 # load default settings from ILCSOFT_CMAKE_MODULES
